### PR TITLE
FHIR-React v3 ("Next") - Integration PR

### PR DESCRIPTION
**Pull request by tony1uphealth**
This PR represents all of the work done for v3 of the FHIR-React library. Significant changes include:

* New Open Source license. No longer MIT licensed, we've chosen "[Creative Commons Attribution Non Commercial Share Alike 4.0 International](https://creativecommons.org/licenses/by-nc-sa/4.0/)"
* Newly updated components

### CODE REVIEW

**Review by liam1uphealth — APPROVED** [review]

**Review by glenn1uphealth — CHANGES_REQUESTED** [review]
New Condition resource has mobile formatting issues. I see also that this PR was not reviewed by 1Up Engineer.
https://github.com/1uphealth/fhir-react/pull/274

 Please correct header formatting issues shown below:
DefVisDSTU2
ExWithSevSTU3
Ex3OfDSTU2

File: package.json [hunk 1]
```diff
@@ -1,12 +1,16 @@
 {
   "name": "fhir-react",
-  "version": "0.2.4",
+  "version": "0.3.0",
```
Comment (tony1uphealth) [inline package.json:3]: This change isn't reflected in the checked-in `package-lock.json`. Please run `npm i` and check in the changes to `package-lock.json`

Comment (tony1uphealth) [reply package.json:3]: Done :)

**Comment by jaceksanko** [comment]
@glenn1uphealth  I corrected the second badge and dates at the mobile view:
Zrzut ekranu 2021-12-10 o 10 06 24

**Review by glenn1uphealth — APPROVED** [review]

**Pull request merged**